apple_passkit/analysis_options.yaml

```diff
@@ -1,5 +1,19 @@
 include: package:flutter_lints/flutter.yaml
```

Comment:
I admit I'm biased on this but have you considered [very_good_analysis](https://pub.dev/packages/very_good_analysis) based on all the additional lints that we're adding?

Reply:
I haven't but I also don't mind you introducing it. I just wanted to have some rules :D Does any of the introduced rules collide with the rules from `very_good_analysis`? If not, you can consider this a step towards the introduction of `very_good_analysis` 🤷

Reply:
they won't collide as far as I know, will introduce it on another PR!